src/Controller/Component/PaginatorComponent.php

```diff
@@ -56,6 +55,23 @@ class PaginatorComponent extends Component
     ];
 
     /**
+     * ORM paginator instance.
+     *
+     * @var \Cake\ORM\Paginator
+     */
+    protected $_paginator;
+
+    /**
+     * {@inheritDoc}
+     */
+    public function __construct(ComponentRegistry $registry, array $config = [])
+    {
+        $this->_paginator = new Paginator();
```

Comment:
Could this maybe be injectable, for easy integration of custom/extended paginators?